introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb

```diff
@@ -0,0 +1,1461 @@
+{
```

Comment:
<p>Add badges as described in CONTRIBUTiNG.md </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks! The badges were added as described in CONTRIBUTiNG.md.